src/Explorer/Notebook/NotebookRenderer/NotebookRenderer.tsx

```diff
@@ -202,7 +203,8 @@ export const makeMapStateToProps = (
   return mapStateToProps;
 };
 
-const makeMapDispatchToProps = (initialDispatch: Dispatch, initialProps: NotebookRendererBaseProps) => {
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const makeMapDispatchToProps = (_initialDispatch: Dispatch, _initialProps: NotebookRendererBaseProps) => {
```

Comment:
why are we changing the param names?

Reply:
IntialDispatch and IntialProps are unused.
So to avoid eslint error  `'initialDispatch' is declared but its value is never read.ts` . Added _ 
<img width="528" alt="2021-10-06" src="https://user-images.githubusercontent.com/79906609/136185469-1f506ae4-e9d0-4632-b5bb-a33810850409.png">